model/term_def_retrieval/retrieval.py

```diff
@@ -0,0 +1,51 @@
+import json
+import os
+
+"""
+Notes for me:
+- Load JSON with each call
+- Each call retrieves a single term or definition
+- Function takes a str
+"""
+
+
+
+
+def get_definition(sample: str, term_to_cui_file: str = "../../data/term_to_cui.json", 
```

Comment:
Is it possible to change these paths to absolute ones instead of relative ones? That way if things were to move around we wouldn't have to worry about maintaining this.